services/settings/README.md

```diff
@@ -83,3 +83,365 @@ The `OCIS_DEFAULT_LANGUAGE` setting impacts the `notification` and `userlog` ser
 *   If  `OCIS_DEFAULT_LANGUAGE` **is set**, the expected behavior is:
     *   The `notification` and `userlog` services and the WebUI use `OCIS_DEFAULT_LANGUAGE`  by default until a user sets another language in the WebUI via _Account -> Language_.
     *   If a user sets another language in the WebUI in _Account -> Language_, the `notification` and `userlog` services and WebUI use the language defined by the user. If no translation is found, it falls back to `OCIS_DEFAULT_LANGUAGE` and then to English.
+
+## Custom Roles
+
+It is possible to replace the default ocis roles (`admin`, `user`) with custom roles that contain custom permissions. One can set `SETTINGS_BUNDLES_PATH` to the path of a `json` file containing the new roles.
+
+Role Example:
+```json
+[
+    {
+        "id": "38071a68-456a-4553-846a-fa67bf5596cc", // ID of the role. Use a random uuid
+        "name": "user-light",                         // Internal name of the role.
```

Comment:
We might need some information about the usage of this internal name. Right now, it seems any random string could fit. I could use "aaaaaaaaaaa" or "Ax30" or "11111".